scene/animation/animation_cache.cpp

```diff
@@ -80,8 +80,7 @@ void AnimationCache::_update_cache() {
 		if (!node) {
 
 			path_cache.push_back(Path());
-			ERR_EXPLAIN("Invalid Track Path in Animation: " + np);
-			ERR_CONTINUE(!node);
+			ERR_CONTINUE_MSG(!node, "Invalid Track Path in Animation: " + np + ".");
```

Comment:
I'd lowercase "track path in animation" for style consistency, but it's a nitpick, I don't mind if it stays as is :)